backend/internal/server/apis/stats/goals.go

```diff
@@ -0,0 +1,46 @@
+package stats
+
+import (
+	"github.com/gofiber/fiber/v2"
+	"github.com/jak103/powerplay/internal/server/apis"
+	"github.com/jak103/powerplay/internal/server/services/auth"
+	"github.com/jak103/powerplay/internal/utils/responder"
+	"github.com/jak103/powerplay/internal/models"
+	"github.com/jak103/powerplay/internal/db"
+	"github.com/jak103/powerplay/internal/utils/locals"
+)
+
+
+func init() {
+	apis.RegisterHandler(fiber.MethodPost, "/goals", auth.Public, postGoalsHandler)
+}
+
+func postGoalsHandler(c *fiber.Ctx) error {
+	log := locals.Logger(c)
+	log.Info("Handling creating new goal")
```

Comment:
We don't need this first log.Info, Dr. Christensen said the logger does this automatically

Reply:
Fixed